### PR TITLE
Simplify the xattr helper method's code with modern Objective-C syntax

### PR DESCRIPTION
**Pull request by huangboju**
### New Pull Request Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar pull request in the [project](https://github.com/rs/SDWebImage/pulls) and found none

* [x] I have updated this branch with the latest master to avoid conflicts (via merge from master or rebase)
* [x] I have added the required tests to prove the fix/feature I am adding
* [x] I have updated the documentation (if necessary)
* [x] I have run the tests and they pass
* [x] I have run the lint and it passes (`pod lib lint`)

This merge request fixes / refers to the following issues: ...

### Pull Request Description

...

### CODE REVIEW

File: SDWebImage/Private/SDFileAttributeHelper.m [hunk 1]
```diff
@@ -95,15 +95,15 @@ + (BOOL)setExtendedAttribute:(NSString *)name value:(NSData *)value atPath:(NSSt
...
-                      [NSNumber numberWithUnsignedInteger:[value length]], @":value.length",
-                      path, @":path",
-                      [NSNumber numberWithBool:follow], @":traverseLink",
-                      [NSNumber numberWithBool:overwrite], @":overwrite",
-                      nil]
+                     @{
+                         @"error": [NSString stringWithUTF8String:strerror(errno)],
+                         @"function": @"setxattr",
+                         @":name": name,
+                         @":value.length": [NSNumber numberWithUnsignedInteger:[value length]],
```
Comment (dreampiggy) [inline SDWebImage/Private/SDFileAttributeHelper.m:102]: Change this to `@(value)` as well.

Comment (huangboju) [reply SDWebImage/Private/SDFileAttributeHelper.m]: solved

Comment (dreampiggy) [reply SDWebImage/Private/SDFileAttributeHelper.m]: OK

**Comment by dreampiggy** [comment]
Thanks for this code style gardening.

The original of this code comes from https://gist.github.com/zydeco/6292773, so we don't write them and just copy to use. But anyway, this code style gardening is OK and looks more matching the other files in SDWebImage.

**Pull request merged**